db/schema.rb

```diff
@@ -97,7 +97,8 @@
     t.string "outgoing_url"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.boolean "hide_input_for_bot_conversations", default: false
```

Comment:
This is used in the frontend, we might need to evaluate the impact.

Reply:
remove the frontend code base on this.